src/Model/Post.php

```diff
@@ -502,11 +502,16 @@ protected function init() {
 					},
 					'capability' => isset( $this->post_type_object->cap->edit_posts ) ? $this->post_type_object->cap->edit_posts : 'edit_posts',
 				],
-				'excerptRendered'           => function () {
+				'excerptRendered' => function () {
 					$excerpt = ! empty( $this->data->post_excerpt ) ? $this->data->post_excerpt : null;
```

Comment:
My understanding is that we want to apply the filters no matter what (so WordPress for example can populate an empty excerpt), and it seems `get_the_excerpt` [also expects a string](https://developer.wordpress.org/reference/functions/get_the_excerpt/).

I think if we just change `$excerpt` to fall back to an empty `''` instead of `null` and revert everything else, we can fix this bug and leave intended behavior intact.


I.e.
```suggestion
					$excerpt = ! empty( $this->data->post_excerpt ) ? $this->data->post_excerpt : '';
```

Thoughts @izzygld ?

Reply:
yes! even better idea.

thsanks @justlevine 

made a [pr for you.](pr - https://github.com/wp-graphql/wp-graphql/pull/2897)

Reply:
Looks great! When I have a minute (unless you beat me to it) I'll copy over the PR title and description and mark it for review.